indexing-service/src/main/java/org/apache/druid/indexing/worker/http/TaskManagementResource.java

```diff
@@ -192,7 +196,8 @@ public void onFailure(Throwable th)
               log.debug(ex, "Request timed out or closed already.");
             }
           }
-        }
+        },
```

Comment:
Some weird asynchrony going on in this method (at least it looks so), so it demands more comments